tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -129,7 +129,7 @@ def validate_output(self, result, cert_data):
         assert cert_data['success_message'] in result.stdout
 
         # validate correct installer used
-        pattern, count = 'satellite-installer --scenario satellite', 2
+        pattern, count = 'foreman-installer --scenario satellite', 2
```

Comment:
I guess, this should not be changed for this test, as it checks first if the right installer is used, and the upstream installer is not used, @omkarkhatavkar can confirm?

Reply:
It should be `satellite-installer`

Reply:
good catch thanks :)